libclc/generic/libspirv/integer/upsample.cl

```diff
@@ -1,46 +1,40 @@
-//===----------------------------------------------------------------------===//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-//===----------------------------------------------------------------------===//
+#include <clc/clc.h>
```

Comment:
Shouldn't we keep the header with license information?

Reply:
oh damn, yes. I'll fix that

Reply:
fixed